src/modules/newTask.js

```diff
@@ -0,0 +1,16 @@
+// eslint-disable-next-line import/extensions
```

Comment:
Your project looks great. As much as possible, kindly avoid disabling linter checks in line. They represent coding standards you share with your team and for most of the time enforce adherence to best coding practices.